Sources/Features/Client/Model/AttachmentBookmarkHandler.swift

```diff
@@ -10,8 +10,9 @@ enum AttachmentBookmarkHandlerImpl: AttachmentBookmarkHandler {
         var attachmentsBookmarksDict = Bookmarks()
         for attachment in attachments {
             do {
-                let bookmark = try attachment.value.bookmarkData(options: URL.BookmarkCreationOptions.minimalBookmark)
-                attachmentsBookmarksDict[attachment.key] = bookmark
+                let bookmark = try attachment.bookmarkData(options: URL.BookmarkCreationOptions.minimalBookmark)
+                let fileName = attachment.lastPathComponent
+                attachmentsBookmarksDict[fileName] = bookmark
```

Comment:
Please consider using the entire path for a key. Otherwise, some keys might get overridden, e.g.:
 - `foo/attachment.log`
 - `bar/attachment.log`

Reply:
That's a good point...however currently we back up the file attachments to a different directory based on just their filename. We would also have to go back there and disambiguate the copied file attachments as well. This is definitely something we can improve and I will create a ticket for this internally. Until then, we will have to make it clear to users we cannot send two file attachments with the same filename using this method.

Reply:
But until then it's still better to make this change, I will do it!

Reply:
Ok, so I'll mark it as resolved then.